src/actions/calculatePoolAnnualPercentageRate.ts

```diff
@@ -71,23 +73,32 @@ export const calculatePoolAnnualPercentageRate = async (
     return apr * 100; // as percentage
   }
 
-  const wildToken = await getWildToken(config.provider);
-  const wEthToken = await getWethToken(config.provider);
-  const lpToken = await getLpToken(config.provider);
+  const tokenPromises = [
+    getWildToken(config.provider),
+    getWethToken(config.provider),
+    getLpToken(config.provider),
+  ];
+
+  const [wildToken, wethToken, lpToken] = await Promise.all(tokenPromises);
```

Comment:
Could combine the `pricePromises and `tokenPromises`

Reply:
You technically could, but it complicates typing in accepting values because all the promises don't return the same things, so they're all `ethers.BigNumber | ethers.Contract | ...` depending on what's all in that array.